docs/gotchas.rst

```diff
@@ -232,6 +232,18 @@ Here is an example of that:
     >>> crs_epsg == crs_deprecated
     False
 
+
+SQLite error on SELECT
```

Comment:
Maybe add something like "Internal Proj Error: ...  SQLite errror on SELECT"

(as it is often only far in the error message that you see this SQLite error on SELECT)

Reply:
Good idea, just updated.